### PR TITLE
Single question view hotfix

### PR DESCRIPTION
**Pull request by abundis28**
# What does this PR do?
- Adds link to logo button to allow the user to return to full forum if the user is in single question view.

**Related Issues**
- None

---
**Merge Checklist**
- [X] Branch is up to date
- [ ] No code style issues

### CODE REVIEW

File: src/main/webapp/question.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <!-- Navbar -->
     <nav class="navbar navbar-expand-lg navbar-light bg-light">
       <!-- Logo -->
-      <a class="navbar-brand" href="#">
+      <a class="navbar-brand" href="index.html">
```
Comment (shaargtz) [inline src/main/webapp/question.html:26]: shouldn't it be "/index.html" ?

Comment (OUMontiel) [reply src/main/webapp/question.html:26]: It's in the same folder as index.html, so it doesn't matter, but we could do it just to be safe. But if we do, we must do the same to signup.html.

Comment (abundis28) [reply src/main/webapp/question.html:26]: It works fine without the dash but whatever you guys think it's best.

Comment (shaargtz) [reply src/main/webapp/question.html:26]: if it works it's ok

**Pull request merged**